libknet/tests/knet_bench.c

```diff
@@ -56,6 +56,7 @@ static void print_help(void)
 	printf(" -c [implementation]:[crypto]:[hashing]    crypto configuration. (default disabled)\n");
 	printf("                                           Example: -c nss:aes128:sha1\n");
 	printf(" -p [active|passive|rr]                    (default: passive)\n");
+	printf(" -P [udp|sctp|tcp]                         (default: udp) protocol (transport) to use\n");
```

Comment:
Not urgent at all, but i would like to be able to set the protocol per link in the -n string, so we can test multiple protocols at once. -P can stay to set the default for all links.